price_calculator/src/main/kotlin/pl/edu/pg/rsww/price_calculator/Geolocation.kt

```diff
@@ -0,0 +1,6 @@
+package pl.edu.pg.rsww.pricecalculator
+
+public class Geolocation(lat: String, long: String) {

```

Comment:
Could this accept `Float` directly and the parsing be moved to the caller since it's the one parsing the input? The other usage of the `Geolocation` class (currently just tests) would benefit from such change.